### PR TITLE
change requirements for ilastik-launch

### PR DESCRIPTION
**Pull request by lldelisle**
Hi,
I tried to install through conda ilastik-launch and I got an error as `${PREFIX}/bin/ilastik-app` does not exists.
This binary only exists if you install ilastik, therefore, I think it should be a requirement.
Regards,
Lucille

### CODE REVIEW

File: recipes/ilastik-launch/meta.yaml [hunk 1]
```diff
@@ -3,9 +3,8 @@ package:
...
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   run:
-    - ilastik-core
-    - volumina
+    - ilastik
```
Comment (k-dominik) [inline recipes/ilastik-launch/meta.yaml]: ideally the minumum requirement would be `>=1.4.0b29` if you choose to also change the app name in the run_ilastik.sh...

**Review by k-dominik — COMMENTED** [review]
Hey @lldelisle,

thank you so much for fixing this! Also we would've run into problems soon, after merging https://github.com/ilastik/ilastik/pull/2582 . We didn't rename the app here in ilastik launch. Since it would be a tiny change, do you think you could make `1.4.0b29` a minimum version for ilastik in this recipe and also change the `run_ilastik.sh` to point to `ilastik` instead of `ilastik-app`?

https://github.com/ilastik/ilastik-conda-recipes/blob/4b3593d5844e07325237149201078a6330952039/recipes/ilastik-launch/run_ilastik.sh#L68

If you don't feel like it it's also fine.

**Comment by lldelisle** [comment]
I fixed the version and binary.
Could you tell me if you want me to change anything else?
Especially the README.

File: recipes/ilastik-launch/meta.yaml (hunk 1 above)
Comment (lldelisle) [reply recipes/ilastik-launch/meta.yaml]: done

File: recipes/ilastik-launch/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ilastik-launch
+
+This allows to get a binary: `${CONDA_PREFIX}/run_ilastik.sh`
```
Comment (k-dominik) [inline recipes/ilastik-launch/README.md]:
```suggestion


This allows to get a binary: `${CONDA_PREFIX}/run_ilastik.sh`.
Intended to be used only in the ilastik binary distribution on Linux/OSX.
```

**Comment by k-dominik** [comment]
thanks a lot @lldelisle !!

**Comment by lldelisle** [comment]
You're welcome. I am happy to help

**Pull request merged**